gdnative-core/src/core_types/access.rs

```diff
@@ -32,15 +32,24 @@ where
 /// Trait for array access guards
 #[doc(hidden)]
```

Comment:
I think they should no longer be hidden since they're sealed now? Just hide all the items instead, and note in the documentation that the trait has no public interface.